src/modules/ekf2/EKF/ekf.h

```diff
@@ -191,6 +191,10 @@ class Ekf final : public EstimatorInterface
 	void getHaglRateInnovVar(float &hagl_rate_innov_var) const { hagl_rate_innov_var = _rng_consistency_check.getInnovVar(); }
 	void getHaglRateInnovRatio(float &hagl_rate_innov_ratio) const { hagl_rate_innov_ratio = _rng_consistency_check.getSignedTestRatioLpf(); }
 
+	void getGravityInnov(float grav_innov[3]) const { memcpy(grav_innov, _aid_src_gravity.innovation, sizeof(_aid_src_gravity.innovation)); }
```

Comment:
It's copying from a float array, but otherwise I agree. We can probably remove these particular getters entirely soon, but for now it's added for consistentcy.

https://github.com/PX4/PX4-Autopilot/blob/deb6053d5670dbf119287b03489cab1e7e9a1876/msg/EstimatorAidSource3d.msg#L13